SwiftReflector/CustomSwiftCompiler.cs

```diff
@@ -44,6 +44,8 @@ public FileSystemModuleLoader (IEnumerable<string> locations)
 
 		public bool Load (string moduleName, TypeDatabase into)
 		{
+			if (moduleName == "Cocoa")
+				moduleName = "AppKit";
```

Comment:
`AppKit` is macOS specific
Do you get `Cocoa` for iOS or other OS ?
What are those symbols that refers to `Cocoa` ?

Reply:
I don't know if that code can know the current platform. I'll open up an issue to look into it later. It's not so much that the symbol is iOS or OS but that the swift code contains:
```
import Cocoa
```
which AFAIK isn't a real namespace? This particular code is compiler for macOS though.